src/components/scenes/GettingStartedScene.tsx

```diff
@@ -211,7 +211,7 @@ export const GettingStartedScene = (props: Props) => {
             })}
           </HeroContainer>
           <Pagination>
-            {Array.from({ length: paginationCount + 1 }).map((_, index) => (
+            {Array.from({ length: paginationCount }).map((_, index) => (
```

Comment:
If `paginationCount` always was `+1` more then `sections.length` then why is it incremented here? Wouldn't this mean we had two extra pagination indicators when we only needed one?